lib/tournament.json

```diff
@@ -625,6 +625,44 @@
       }
     ]
   ],
+  "rankings": [
```

Comment:
Btw this is not generated manually, I have a script that generates the data but I'll check it in later to focus on delivering the data for now.